### PR TITLE
Add envfunc to export kind cluster logs

### PR DESCRIPTION
**Pull request by alexandrevilain**

### CODE REVIEW

**Comment by ShwethaKumbla** [comment]
@alexandrevilain Thanks for your contribution.
If you could update the usage of this function under examples/kind that would be great.

File: pkg/envfuncs/kind_funcs.go [hunk 1]
```diff
@@ -205,3 +205,26 @@ func LoadImageArchiveToCluster(name, imageArchive string) env.Func {
 		return ctx, nil
 	}
 }
+
+// ExportKindClusterLogs returns an EnvFunc that
+// retrieves a previously saved kind Cluster in the context (using the name), and then export cluster logs
+// in the provided destination.
+func ExportKindClusterLogs(name, dest string) env.Func {
```
Comment (ShwethaKumbla) [inline pkg/envfuncs/kind_funcs.go:212]: If you could update the usage of this function under examples/kind that would be great.

Comment (alexandrevilain) [reply pkg/envfuncs/kind_funcs.go:212]: Hi! Thanks! Example is now updated !

**Comment by alexandrevilain** [comment]
Failed test seems not related to my PR:
```
    helm_test.go:68: failed waiting for the Deployment to reach a ready state
=== RUN   TestHelmChartRepoWorkflow/Repo_based_helm_chart_workflow/run_Chart_Tests
--- FAIL: TestHelmChartRepoWorkflow (302.77s)
    --- FAIL: TestHelmChartRepoWorkflow/Repo_based_helm_chart_workflow (302.77s)
        --- FAIL: TestHelmChartRepoWorkflow/Repo_based_helm_chart_workflow/Deployment_is_running_successfully (300.01s)
        --- PASS: TestHelmChartRepoWorkflow/Repo_based_helm_chart_workflow/run_Chart_Tests (0.13s)
```

**Review by vladimirvivien — COMMENTED** [review]
/lgtm

**Pull request merged**